src/Database/Schema/MysqlSchemaDialect.php

```diff
@@ -67,7 +67,7 @@ public function describeOptionsSql(string $tableName, array $config): array
     /**
      * @inheritDoc
      */
-    public function convertOptionsDescription(TableSchema $schema, array $row): void
+    public function convertOptionsDescription(TableSchemaInterface $schema, array $row): void
```

Comment:
You need to revert these signature changes. We cannot change method signatures in minor releases as it would be a BC break.

Reply:
Alright, will do.